.github/workflows/advance-zed.yml

```
@@ -79,22 +79,22 @@ jobs:
           go-version: "1.16"
       - uses: actions/setup-node@v2
         with:
-          cache: npm
+          cache: yarn
           node-version-file: .nvmrc
-      - run: npm install https://github.com/brimdata/zed#${{ steps.zed_pr.outputs.sha }}
-      - run: npm install --no-audit
-      - run: npm run build
-      - run: npm run lint
-      - run: npm run tsc
-      - run: npm test -- --maxWorkers=2 --ci
-      - run: npm run test:api
-      - run: npm run test:system
+      - run: yarn install https://github.com/brimdata/zed#${{ steps.zed_pr.outputs.sha }}
+      - run: yarn
```

Adding `--inline-builds` here prints build output for dependencies, which is really helpful when one fails.
```suggestion
      - run: yarn --inline-builds
```